test/v1_0_2/document.js

```diff
@@ -22,10 +22,13 @@
      */
     function sendRequest(type, url, params, body, expect) {
         var reqUrl = params ? (url + '?' + qs.stringify(params)) : url;
-        var pre = request[type](reqUrl)
-            .set('X-Experience-API-Version', '1.0.1');
+
+        var headers = helper.addAllHeaders({});
+        var pre = request[type](reqUrl);
         if (body) {
```

Comment:
This will not attach the header if issuing a GET request since GET does not specify a body